### ISSUE TITLE
Hover circle of the close button is not symmetrical to the button icon

### ISSUE THREAD

**Issue by bieniu** [home-assistant/frontend#6466]
## Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

## The problem

In the more-info window, the hover circle of the close button is not symmetrical to the button icon.

![hover](https://user-images.githubusercontent.com/478555/88477967-d23a8e00-cf44-11ea-9402-a711fab4f730.gif)

## Expected behavior

The hover circle should be symmetrical to the button icon.

## Environment

- Home Assistant release with the issue: 0.113.x
- Last working Home Assistant release (if known): 0.112.x
- Browser and browser version: Opera 69.0/Edge 84.0
- Operating system: Windows/Debian

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

None

**Comment by SeanPM5** [comment]
Duplicate of #6285

Bram fixed it already, it is just waiting on Google to release.
